src/routes/project/controllers/acceptInvite.ts

```diff
@@ -12,21 +13,26 @@ export default async (ctx: Context, next: Next): Promise<void> => {
     const bytes = CryptoJS.AES.decrypt(result.key, process.env.INVITE_SECRET_KEY as string);
     const decryptedData = JSON.parse(bytes.toString(CryptoJS.enc.Utf8));
     const project = await Project.findOne({ _id: decryptedData.projectId });
+    const user = (await User.findOne({ _id: ctx.state.user._id })) as IUserDocument;
     if (!project) {
-      ctx.throw('something error');
-      // return;
+      ctx.response.status = 404;
+      ctx.throw('project not found');
```

Comment:
ctx.throw(404, 'project not found')로 쓸 수 있을것 같습니다!
```suggestion
      ctx.throw(404, 'project not found')
```